RELEASE-NOTES.md

```diff
@@ -44,6 +44,9 @@
 **EUI-2744** EventToComplexTypes regression with hidden fields
 **EUI-4106** Show/hide functionality
 
+### Version 3.1.6-case-field-labels-showing-on-summary-screen-when-complex-type-is-hidden
```

Comment:
How is this 4.2.0 if the only thing we're merging in is something from `3.1.6`?

Reply:
Updated the release candidates